Library/AppEnvironment.swift

```diff
@@ -114,6 +121,7 @@ public struct AppEnvironment {
     koala: Koala = AppEnvironment.current.koala,
     language: Language = AppEnvironment.current.language,
     launchedCountries: LaunchedCountries = AppEnvironment.current.launchedCountries,
+    liveAuthToken: String? = AppEnvironment.current.liveAuthToken,
```

Comment:
Ok, this will change along with https://github.com/kickstarter/kickstarter/pull/6045